packages/api/src/providers/fitbit.ts

```diff
@@ -332,4 +336,51 @@ export class Fitbit extends Provider implements OAuth2 {
       }
     );
   }
+
+  async postAuth(token: string, userId?: string) {
+    const accessToken = JSON.parse(token).access_token;
+
+    const scopes = JSON.parse(token).scope;
+
+    const subscriptionTypes: Record<FitbitScopes, string> = {
```

Comment:
If I understand correctly, instead of `string` we could use `Omit<FitbitCollectionTypes, "userRevokedAccess">`, right?